cmd/task/task.go

```diff
@@ -28,12 +28,14 @@ Example: 'task hello' with the following 'Taskfile.yml' file will generate an
 'output.txt' file with the content "hello".
 
 '''
-hello:
-  cmds:
-    - echo "I am going to write a file named 'output.txt' now."
-    - echo "hello" > output.txt
-  generates:
-    - output.txt
+version: '2'
```

Comment:
Actually, this should be version 3 since this PR is targeting the `v3` branch 🙂 

Good catch, though. This example was forgotten for a long time.

```suggestion
version: '3'
```